lighthouse-cli/performance-experiment/formatters/partials/config-panel.html

```diff
@@ -0,0 +1,286 @@
+<style>
+.config-panel {
+  padding: 0 0 0 0;
+  background: #FAFAFA;
+}
+
+.config-panel__header {
+  height: 50px;
+  border-top: 1px solid #EBEBEB;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+  padding-right: 30px;
+}
+
+.config-panel__body {
+  border-top: 1px solid #EBEBEB;
+  height: 300px;
+  display: none;
+}
+
+.config-panel.expanded .config-panel__body {
+  display: block;
+}
+
+.config-panel__message {
+  flex: 1 1 0;
+  padding: 0 10px;
+}
+
+.config-panel__panel-toggle {
+  background-image: url('data:image/svg+xml;utf8,<svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8l-6 6 1.41 1.41L12 10.83l4.59 4.58L18 14z"/><path d="M0 0h24v24H0z" fill="none"/></svg>');
+  width: 24px;
+  height: 24px;
+  border: none;
+  flex: 0 0 auto;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: contain;
+  background-color: transparent;
+  margin-left: 8px;
+}
+
+.config-panel.expanded .config-panel__panel-toggle {
+  background-image: url('data:image/svg+xml;utf8,<svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"/><path d="M0 0h24v24H0z" fill="none"/></svg>');
```

Comment:
FF has issues parsing svg data urls with `#` in them. Can you use `fill="black"` for these colors?